### ISSUE TITLE
Encoding problem on Unpacking text content

### ISSUE THREAD

**Issue by yarongon** [chrismattmann/tika-python#169]
When unpacking a resource, if the content in UTF-8 a `UnicodeDecodeError` exception is raised. Exception occurs on [`unpack.py`](https://github.com/chrismattmann/tika-python/blob/master/tika/unpack.py) on line 99:
```python
content = _text_wrapper(tarFile.extractfile(contentMember)).read()
```
I was able to fix it using the encoding argument:
```python
_text_wrapper(tarFile.extractfile(contentMember), encoding='utf8').read()
```

Can you please add this to the code?

**Comment by yarongon** [comment]
Hi,
I've added the code with some tests and sent a PR.

**Comment by chrismattmann** [comment]
Fixed in #171
